LevelUp.Pos.ProposedOrders.Tests/SplitTenderTests.cs

```diff
@@ -295,19 +304,24 @@ public void SplitTenderExample_LevelUp_Second_Discount_Remaining_Balance()
                 $"Expected: {expectedProposedOrderValues}" + Environment.NewLine +
                 $"Actual: {proposedOrderValues}");
 
-            // available discount amount $10.00
-            int availableDiscountAmount = 1000;
+            // available discount amount $5.30
+            int availableDiscountAmount = 530;
 
             // POS applies $10.00 pretax discount to check and updates subtotal and tax
             check.ApplyDiscount(availableDiscountAmount);
 
+            // $ 4.70 (Subtotal Remaining)
+            // $ 0.28 (Tax)
+            // $ 4.98 (Total)
+            // $10.28 (Total + Discount Applied) 
```

Comment:
I'm hoping that breaking it down like this adds clarity.  I would like to see this for any unit tests going forward.